site-src/geps/gep-1282.md

```diff
@@ -0,0 +1,79 @@
+# GEP-696: GEP template
```

Comment:
We would appear to want _some kind of title for this_?

Reply:
🤦‍♂️